### ISSUE TITLE
batch_norm parameters issue

### ISSUE THREAD

**Issue by konioyxgq** [HuangYG123/CurricularFace#15]
On line 104 of train.py, you write "separate batch_norm parameters from others; do not do weight decay for batch_norm parameters to improve the generalizability"(https://github.com/HuangYG123/CurricularFace/blob/8b2f47318117995aa05490c05b455b113489917e/train.py#L104). Is this your conclusion? Or is the conclusion of a certain paper? Can you explain the reason for this?

**Comment by HuangYG123** [comment]
We follow this setting as https://github.com/ZhaoJ9014/face.evoLVe.PyTorch.

**Comment by konioyxgq** [comment]
Okay thank you. What do you think is the reason for this?
